### ISSUE TITLE
editorial elements affect layers

### ISSUE THREAD

**Issue by rettinghaus** [rism-ch/verovio#618]
Editorial elements within `staff` are recognized as `layer`. 

![annot-vs-layer](https://user-images.githubusercontent.com/7693447/27001457-f8b93a26-4dca-11e7-9d8e-525ee3768353.png)

```xml
<?xml version="1.0" encoding="UTF-8"?>
<?xml-model href="http://music-encoding.org/schema/3.0.0/mei-all.rng" type="application/xml" schematypens="http://relaxng.org/ns/structure/1.0"?>
<?xml-model href="http://music-encoding.org/schema/3.0.0/mei-all.rng" type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?>
<mei xmlns="http://www.music-encoding.org/ns/mei" meiversion="3.0.0">
  <meiHead>
    <fileDesc>
      <titleStmt>
        <title />
      </titleStmt>
    </fileDesc>
  </meiHead>
  <music>
    <body>
      <mdiv>
        <score>
          <scoreDef meter.count="3" meter.unit="4">
            <staffGrp>
              <staffDef n="1" lines="5" clef.shape="C" clef.line="3"/>
            </staffGrp>
          </scoreDef>
          <section>
            <measure>
              <staff n="1">
                <layer n="1">
                  <note dur="4" oct="3" pname="a" />
                  <note dur="4" oct="4" pname="e" />
                  <rest dur="4" />
                </layer>
              </staff>
            </measure>
            <measure>
              <staff n="1">
                <annot>important annotation</annot>
                <layer n="1">
                  <note dur="4" oct="3" pname="a" />
                  <note dur="4" oct="4" pname="e" />
                  <rest dur="4" />
                </layer>
              </staff>
            </measure>
            <measure>
              <staff n="1">
                <layer n="1">
                  <note dur="4" oct="3" pname="a" />
                  <note dur="4" oct="4" pname="e" />
                  <rest dur="4" />
                </layer>
                <annot>less important annotation</annot>
              </staff>
            </measure>
          </section>
        </score>
      </mdiv>
    </body>
  </music>
</mei>
```

[This is just a reminder.]

**Comment by lpugin** [comment]
Sorry, I do not understand it. Could you clarify?

**Comment by craigsapp** [comment]
The `<annot>` text is forcing layout of the single layer as either layer 1 of a two-layer staff or layer 2 of a two-layer staff, depending on its position before or after the single `<layer>` element of the staff.

**Comment by lpugin** [comment]
OK, got it.

**Comment by lpugin** [comment]
To be solved with #262

**Comment by lpugin** [comment]
Improved here https://mail.google.com/mail/u/0/#inbox/15cd97f26e502ad0
